### ISSUE TITLE
SyntaxError: Escape sequence not recognized: {\textbar}

### ISSUE THREAD

**Issue by phiresky** [larsgw/citation.js#194]
```
@misc{noauthor_zotero_nodate,
	title = {Zotero {\textbar} {Your} personal research assistant},
	url = {https://www.zotero.org/},
	abstract = {Zotero is a free, easy-to-use tool to help you collect, organize, cite, and share research.},
	urldate = {2019-12-12},
	journal = {www.zotero.org}
}
```

extraction error SyntaxError: Escape sequence not recognized: {\textbar}

that bibtex is generated by Zotero, see https://en.wikipedia.org/api/rest_v1/data/citation/bibtex/https%3A%2F%2Fwww.zotero.org%2F

**Comment by phiresky** [comment]
looks like this is a simple addition to https://github.com/citation-js/citation-js/blob/master/packages/plugin-bibtex/src/input/tokens.json ?

**Comment by pussiatoday** [comment]
```
  {\textbar}
+ {\textless}
+ {\textgreater}
```

**Comment by larsgw** [comment]
Sorry for not replying earlier.

> looks like this is a simple addition to https://github.com/citation-js/citation-js/blob/master/packages/plugin-bibtex/src/input/tokens.json ?

That is true for that file, but the [`v0.5` branch](https://github.com/citation-js/citation-js/tree/v0.5/) has some more recent development, including a new parser, that I am trying to finish.

**Comment by larsgw** [comment]
This issue should be fixed in `v0.5.0-alpha.10`. If not, please re-open the issue.
